src/DataDeps.jl

```diff
@@ -38,15 +42,40 @@ Even if that means downloading the dependancy and putting it in there.
 
 This is basically the function the lives behind the string macro `datadep"DepName`.
 """
-function resolve(datadep::AbstractDataDep, calling_filepath)::String
+function resolve(datadep::AbstractDataDep, calling_filepath, inner_filepath)::String
```

Comment:
the docstring looks out of sync now. it says `resolve(datadep)`. would be interesting to have a short description of the other two parameters